src/index.tsx

```diff
@@ -491,7 +491,7 @@ class H5AudioPlayer extends Component<PlayerProps> {
           <button
             key={key}
             aria-label={loop ? 'Enable Loop' : 'Disable Loop'}
-            className="rhap_button-clear rhap_repeat-button"
+            className="rhap_button-clear rhap_repeat-button rhap_loop"
```

Comment:
There's already `rhap_repeat-button`, and we don't need an extra class here, right?
```suggestion
            className="rhap_button-clear rhap_repeat-button"
```

Reply:
I overlooked `rhap_repeat-button`. 😅  I've removed it.